server/datastore/mysql/migrations/tables/20241220114903_ChangeDDMJSONColumnToText.go

```diff
@@ -0,0 +1,25 @@
+package tables
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20241220114903, Down_20241220114903)
+}
+
+func Up_20241220114903(tx *sql.Tx) error {
+	_, err := tx.Exec(`	
+ALTER TABLE mdm_apple_declarations
+    CHANGE raw_json raw_json MEDIUMTEXT COLLATE utf8mb4_unicode_ci NOT NULL -- 16MB max size`)
```

Comment:
In case the user substitutes the secret outside of a string I presume? Could you just add a small comment explaining that change for future reference?

Reply:
Yes, like:

```json
  "Payload": {
    "RequireAlphanumericPasscode": true,
    "MinimumLength": $FLEET_SECRET_PASSCODE_LENGTH,
```